packages/zoe/tools/fakePriceAuthority.js

```diff
@@ -176,6 +173,17 @@ export async function makeFakePriceAuthority(options) {
     return promiseKit.promise;
   }
 
+  async function* generateQuotes(amountIn, brandOut) {
+    let record = await notifier.getUpdateSince();
```

Comment:
Rather than being triggered by the notifier.getUpdateSince call resolving, can we just pipe this in directly to the `wake` function? Is there anything I'm missing about the one to many publications that makes this a bad idea?

Reply:
As above.